tests/e2e-playwright/specs/admin/students/setup-wizard.spec.ts

```diff
@@ -22,13 +22,6 @@ test.describe.serial( 'Setup Wizard @setup', () => {
 		await pluginsPage.goTo( 'admin.php?page=sensei_setup_wizard' );
 	} );
 
-	test( 'opens when first activating the Sensei LMS plugin', async () => {
```

Comment:
I think this test has value and would be good to keep it. Could we update it instead to simulate a new install by clearing out the option?

Reply:
Good idea! Fixed in https://github.com/Automattic/sensei/pull/6562/commits/df7d62cde7b3a0ec7ad4d49f6e745e8a918961c5